src/client/Microsoft.Identity.Client/ApiConfig/Executors/ConfidentialClientExecutor.cs

```diff
@@ -59,7 +62,13 @@ public ConfidentialClientExecutor(IServiceBundle serviceBundle, ConfidentialClie
                 _confidentialClientApplication.AppTokenCacheInternal).ConfigureAwait(false);
        
             requestParams.SendX5C = clientParameters.SendX5C ?? false;
-            
+
+            if (ServiceBundle.Config.UseManagedIdentity)
```

Comment:
nit: `IsManagedIdentityEnabled`

Reply:
UseManagedIdentity is inline with how the other parameters are named.